### ISSUE TITLE
After installing v114, my settings are gone

### ISSUE THREAD

**Issue by BarbacoaUSA** [Rem0o/FanControl.Releases#770]
The app asked if I wanted to update to v114 so I did.  Afterwards, my fan curve and names, etc. were all gone.  Even though I made sure I loaded my correct settings file, it was like I was starting over with configuration.  I had to name my controls and fans again and remake and assign my curves.

This is my first time upgrading this software (I'm a new user).  Is this normal for each update?  If not, what did I do wrong?  I don't want to have to re-create all my settings each time I update the software.

Thanks.

**Comment by MartinTophill** [comment]
This is not normal.
I've been updating this software on every new version ( since ver. 67 ) and it usually takes 5 seconds with no faults.
Never have my settings went disappeared.

I suggest you do a clean install and backup your new settings.

Shit happens...

**Comment by BarbacoaUSA** [comment]
Ok, thanks for the reply.  Fortunately, I did not have a lot of items to recreate.  I'll try the next update and hope for the best.  I'll also make a copy of my settings file first and save it to another location just in case.  Thanks.

**Comment by Rem0o** [comment]
I still don't understand how you can get your settings wipes out. Are your starting the exe from the zip directly? 

Your fancontrol folder should look like this:
![image](https://user-images.githubusercontent.com/14118956/168873157-e94a5bec-d357-43b6-8460-2061a9af1fed.png)

**Comment by trankillity** [comment]
My folder is located in my `%USERPROFILE%\AppData\Local\` folder. I have FanControl set to run on startup. I used the auto-update from 112 to 115. Upon launching 115, my `userConfig.json` file was empty.

**Comment by Rem0o** [comment]
Any log.txt?

**Comment by schniper** [comment]
Just a note: I checked userConfig.json after the update (I lost my settings too) and it was full of nulls. So the file was no 0 in size, but it was written over with empty chars. Sadly I don't have the copy anymore and I since reverted to 112. Thank you for an otherwise great app! :)

**Comment by NiceSk1ll3r** [comment]
Same thing happened to me today, updated from v112 -> v115 and everything is default, tried "load config", but not a single old config comes back unfortunately. Maybe it happened because I ran Updater as admin, it didn't want to update, when I just clicked "update".

**Comment by BarbacoaUSA** [comment]
FWIW, after my initial problem as described in the opening post I later installed v115 when it was available and it worked fine.  I didn't lose any settings and didn't have to reset anything.  I installed it the same way I did v114 (the auto updater that pops up when the app starts).

Don't know why it worked one time but not the other, but it seems all good now, and I made sure to make a backup of my config file just in case for the future.  :-)
